screeps-profiler.js

```diff
@@ -14,20 +14,36 @@ function setupProfiler() {
     profile(duration, filter) {
       setupMemory('profile', duration || 100, filter);
     },
+    background(duration, filter) {
```

Comment:
I think dropping the duration would be best.  People should just be conscious about leaving this enabled.  Could also update the config to have a callback option.

```
profiler.wrap(fn, {
  intervalLength: 20, // 20 would be default
  onInterval: (rawMemory) => {} // Player can do w/e with raw memory.
});
```